lib/pdf.js

```diff
@@ -4,7 +4,9 @@ const mkdirp = require('mkdirp')
 const puppeteer = require('puppeteer')
 
 module.exports = async (opts = {}) => {
-  const browser = await puppeteer.launch()
+  const browser = await puppeteer.launch({
+    args: ['--user-agent=pdf']
```

Comment:
Could we rather do [`page.setUserAgent()`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagesetuseragentuseragent) in l. 11?

Reply:
Yes so we just need to set it for the current page rather the whole instance.